Linux / Windows / macOS-cross all fail to build — the `#[cfg(not(target_os = "macos"))]` variant of `pump_cef_message_loop` was missed in the rename:

```
error[E0308]: mismatched types
    --> apps/buffr/src/main.rs:5040:31
     |
5040 |         pump_cef_message_loop(&mut self.cef_next_pump_at);
     |         --------------------- ^^^^^^^^^^^^^^^^^^^^^^^^^^ expected `&mut bool`, found `&mut Option<Instant>`
     |
note: expected mutable reference `&mut bool`, but found
    --> apps/buffr/src/main.rs:5411:4
     |
5411 | fn pump_cef_message_loop(_pump_active: &mut bool) {
```

The macOS-package job passes because it hits the updated macOS branch; everything else builds against the stale stub.

One-line fix:

```rust
#[cfg(not(target_os = "macos"))]
fn pump_cef_message_loop(_next_pump_at: &mut Option<Instant>) {
    cef::do_message_loop_work();
}
```

--- a/apps/buffr/src/main.rs
+++ b/apps/buffr/src/main.rs
@@ -5408,7 +5408,7 @@ fn pump_cef_message_loop(next_pump_at: &mut Option<Instant>) {
 }
 
 #[cfg(not(target_os = "macos"))]
-fn pump_cef_message_loop(_pump_active: &mut bool) {
+fn pump_cef_message_loop(_next_pump_at: &mut Option<Instant>) {
     cef::do_message_loop_work();
 }
 

